src/test/kotlin/graphics/scenery/tests/examples/volumes/VolumeSamplingExample.kt

```diff
@@ -115,7 +117,7 @@ class VolumeSamplingExample: SceneryBase("Volume Sampling example", 1280, 720) {
             connector.spatial().orientBetweenPoints(p1.spatial().position, p2.spatial().position, true, true)
         }
 
-        val volume = Volume.fromBuffer(emptyList(), volumeSize, volumeSize, volumeSize, UnsignedByteType(), hub)
+        val volume = Volume.fromBuffer(emptyList(), volumeSize, volumeSize, volumeSize, UnsignedShortType(), hub)
```

Comment:
Is the change to UnsignedShortType here correct? Does the example work as intended?

Reply:
Using a procedural example, I can say that the ShortType has much cleaner edges. ByteType looks all fuzzy and has stripy patterns. So I'd tend to say ShortType is the right type here.